### ISSUE TITLE
[Bug] [SuperPins] No more option to stretch overhanging Essentials/"Essentialised" Pins over the whole width of the tabbar?

### ISSUE THREAD

**Issue by antikovt** [JLBlk/Zen-Themes#55]
Didn't there use to be an option to stretch the button of the essential of the last row over its entire width? I feel like there was, but if not, then it's a feature request, hahah

![image](https://github.com/user-attachments/assets/022cf7cc-5db4-4d5d-ac7f-b8844f106012)
![image](https://github.com/user-attachments/assets/0f1163e2-f6ec-409f-b311-da202ee6d8b1)

**Comment by JLBlk** [comment]
Yeah, there wasn't an option for that, but really good nonetheless.
Could be hard to implement though, i'll see when i have the time

**Comment by antikovt** [comment]
> Yeah, there wasn't an option for that

Huh, I could've sworn it did behave like this at one point. Must be tripping.

**Comment by TheBigWazz** [comment]
Would like this functionality in Legacy Pins layout as well. Been trying to fiddle with it today and have come up short.

**Comment by CosmoCreeper** [comment]
Great idea, I'm thinking of implementing it on my end and I can create a pull request if I get it working.

**Comment by CosmoCreeper** [comment]
Just finished! Doesn't work on essentials due to some styling choices built-in to the Zen Browser (so I won't be doing it until further notice), but works on pinned tabs.

**Comment by CosmoCreeper** [comment]
I just got it to work for Essentials (turns out further notice came pretty soon!).
